react/week1/react-w1/src/App.js

```diff
@@ -0,0 +1,11 @@
+
+import './App.css';
+
+function ToDo(props) {
+  return (    
+      <ul><li>{props.description} , {props.endDate}</li></ul> 
+  );
+}
```

Comment:
the `ToDo` component is already used inside a `<ul>` tag, so you don't need to include a `<ul>` tag inside the component 

```suggestion
  return (    
      <li>{props.description} , {props.endDate}</li>
  );
}
```

Reply:
Got it.Thanks

Reply:
Thanks for the feedback